sdk/samples/azidentity/manual-tests/managed-identity/functions/handler.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is awesome, I didn't realize you could write Azure Functions using Go!

Reply:
You can deploy an executable binary as a Function! https://docs.microsoft.com/azure/azure-functions/functions-custom-handlers

Reply:
Now Go is all I need (uninstalls Visual Studio)